lib/qa/authorities/linked_data/find_term.rb

```diff
@@ -130,16 +130,16 @@ def predicates_with_subject_uri(graph, expected_uri) # rubocop:disable Metrics/M
             subj = st.subject.to_s
             next unless subj == expected_uri
             pred = st.predicate.to_s
-            obj  = st.object.to_s
-            next if blank_node? obj
+            obj  = st.object
+            next if obj.anonymous?
             if predicates_hash.key?(pred)
```

Comment:
This is the check for blank nodes from ruby-rdf/rdf gem.  Better to use it directly than wrap it in a separate method that adds nothing.